### DISCUSSION TITLE
Login no iOS não funciona (Vue + ExpressJs + PassportJs)

### DISCUSSION THREAD

**Discussion by ortizvinicius** [frontendbr/forum#1291] [Perguntas e Respostas]
Boa tarde pessoal, 

Tenho uma aplicação web que roda em servidores locais, o front é construído em VueJs e o back em NodeJs.

Tudo estava normal, porém de um tempo pra cá o login parou de funcionar no iOS, sendo que no Android funciona normalmente, até no HTTPS não funciona. No back quem lida com isso é o PassportJs e ExpressJs.

A requisição é feita porém volta pra tela de login, como se não estivesse salvando a sessão.

Alguém sabe se mudou algo em questão de sessão ou algo assim no iOS?

**Comment by willycamargo** [comment]
Esta utilizando alguma api no navegador para "verificar" a sessão do usuário?

Cookie, localstorage etc?

Já tive problemas com isso no iOS antes...
